generators/chipyard/src/main/scala/config/SodorConfigs.scala

```diff
@@ -0,0 +1,47 @@
+package chipyard
+
+import chisel3._
+
+import freechips.rocketchip.config.{Config}
+
+class Sodor1StageConfig extends Config(
+  new sodor.common.WithNSodorCores(1, internalTile = sodor.common.Stage1Factory) ++
```

Comment:
Can you add comments saying what type of core it is here?

```suggestion
  new sodor.common.WithNSodorCores(1, internalTile = sodor.common.Stage1Factory) ++ // 1 sodor single-stage core
```

Reply:
Done.